src/GregClient/Requests/PackageUploadRequestBody.cs

```diff
@@ -23,7 +23,8 @@ public PackageUploadRequestBody()
             string contents, string engine, string engineVersion,
             string metadata, string group, IEnumerable<PackageDependency> dependencies,
             string siteUrl, string repositoryUrl, bool containsBinaries, 
-            IEnumerable<string> nodeLibraryNames, IEnumerable<string> hostDependencies):
+            IEnumerable<string> nodeLibraryNames, IEnumerable<string> hostDependencies,
+            string copyright_holder, string copyright_year) :
```

Comment:
@zeusongit @QilongTang this is an API break - If older clients try to call this API they will get a missing message exception.

I don't have any concrete examples, but it's asking for some trouble.

I think it would be better to obsolete this as was done below...

Reply:
I was assuming `Dynamo and only a single Dynamo version should be compatible with a single greg.dll version` but yeah but looking at the history, we have been obsoleting old ones. @zeusongit Shall we follow the pattern?